static/js/message_lists.js

```diff
@@ -8,11 +8,22 @@ export function set_current(msg_list) {
     current = msg_list;
 }
 
-export function initialize() {
-    home = new message_list.MessageList({
-        table_name: "zhome",
-        filter: new Filter([{operator: "in", operand: "home"}]),
-        excludes_muted_topics: true,
-    });
-    current = home;
```

Comment:
We'll want to effectively revert this hunk, moving the `home` data structure back to this file, for import cycle reasons.

Reply:
Ok, updated!